src/main/java/org/springframework/data/neo4j/core/mapping/SpringDataCypherDsl.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2011-2023 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.data.neo4j.core.mapping;
+
+import org.apiguardian.api.API;
+import org.neo4j.cypherdsl.core.FunctionInvocation;
+import org.neo4j.cypherdsl.core.Functions;
+import org.neo4j.cypherdsl.core.Named;
+import org.neo4j.cypherdsl.core.Node;
+import org.neo4j.cypherdsl.core.Relationship;
+import org.neo4j.cypherdsl.core.renderer.Dialect;
+
+import java.util.function.Function;
+
+/**
+ * Supporting class for CypherDSL related customizations.
+ *
+ * @author Gerrit Meier
+ */
+@API(status = API.Status.INTERNAL)
+public class SpringDataCypherDsl {
```

Comment:
Make it final and add an private default constructor, just for completeness please.